membership-attribute-service/test/controllers/AttributeControllerTest.scala

```diff
@@ -96,7 +96,7 @@ class AttributeControllerTest extends Specification with AfterAll with Idiomatic
   private val newspaperPlusCookie = Cookie("newspaperPlus", "true")
   private val guardianWeeklyCookie = Cookie("guardianWeekly", "true")
   private val tierThreeCookie = Cookie("tierThree", "true")
-  private val guardianLightCookie = Cookie("guardianLight", "true")
+  private val guardianAdLiteCookie = Cookie("guardianAdLite", "true")
```

Comment:
Hmm I'm not sure what this is used for here, but we don't drop a cookie specifically related to Ad-Lite any more, rather a cookie is dropped representing the benefit (`gu_allow_reject_all`).

Reply:
Ok, I've updated the tests file